syft/frameworks/torch/tensors/crypten/syft_crypten.py

```diff
@@ -0,0 +1,52 @@
+import torch
+
+from crypten.mpc import MPCTensor
+
+from syft.generic.frameworks.hook import hook_args
+from syft.generic.tensor import AbstractTensor
+
+from functools import wraps
+from syft.generic.frameworks.hook.trace import tracer
+from syft.generic.frameworks.overload import overloaded
+
+
+class SyftCrypTensor(AbstractTensor):
+    def __init__(
+        self, owner=None, id=None, tensor=None, tags: set = None, description: str = None,
+    ):
+        super().__init__(id=id, owner=owner, tags=tags, description=description)
+        self.tensor = tensor
+
+    def get_class_attributes(self):
+        """
+        Specify all the attributes need to build a wrapper correctly when returning a response,
+        """
+        # TODO: what we should return specific for this one?
+        return {}
+
+    @property
+    def data(self):
+        return self
+
+    @data.setter
+    def data(self, new_data):
+        self.tensor = new_data.child
+        return self
+
+    @tracer(method_name="add")
```

Comment:
so do we need this one, or is the add call directly forwarded to the child?

Reply:
It is forwarded to the child (in our case we do not have a child).

But, if I change in the ```__init__``` method to have this:
```
self.child = tensor
```
Then when building the plan, the traces look like this:
```
-> for log in sy.hook.trace.logs:
(Pdb) p sy.hook.trace.logs
[(('crypten.load', None, ('crypten_data', 1), {}), SyftCrypTensor>tensor(0.)), (('crypten.load', None, ('crypten_data', 2), {}), SyftCrypTensor>tensor(0.)), (('__add__', tensor(0.), (tensor(0.),), {}), tensor(0.)), (('get_plain_text', SyftCrypTensor>tensor(0.), (), {}), SyftCrypTensor>None)]
(Pdb)
```
And we get an error in the plan with:
```
 ValueError: The following tensor was used but is not known in this plan: 
E                   0.0
E                   Possible reasons for this can be:
E                   - This tensor is external to the plan and should be provided using the state. See more about plan.state to fix this.```

This is because (I think) the values for the add operations are not wrapped in SyftCrypTensor. The solution (there might be another) was to trace the ```add``` operation in the ```SyftCryptensor``` class.